### DISCUSSION TITLE
Error; add g92 e0 to layer code

### DISCUSSION THREAD

**Discussion by sandy1man** [OrcaSlicer/OrcaSlicer#6389] [Q&A]
I feel like a elephant on roller skates.  I'm using DesignSpark to create 3D models and Orca slicer for the post processor.   I am getting an error code that says I must add G92 E0 to the layer Gcode.   And I am lost a goose....

**Comment by matthiasPlank** [comment]
Do you find a solution? I have the same Problem!
